.github/workflows/docker-openmetadata-airflow.yml

```diff
@@ -40,5 +44,5 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name == 'release' }}
           # Update tags before every release
-          tags: 'openmetadata/airflow:0.10.1,openmetadata/airflow:latest'
+          tags: 'openmetadata/airflow:${{ github.event.inputs.tag }},openmetadata/airflow:latest'
```

Comment:
`github.event.inputs.tag` will be null in case of Github CI triggered by Release published event!